algos.yaml

```diff
@@ -352,6 +471,47 @@ bigann-1B:
                 "nprobe=256,quantizer_efSearch=64",
                 "nprobe=256,quantizer_efSearch=128",
                 "nprobe=256,quantizer_efSearch=512"] 
+    team11:
+      docker-tag: billion-scale-benchmark-faissconda
+      module: benchmark.algorithms.faiss_t1_plus
+      constructor: Faiss
+      base-args: ["@metric"]
+      run-groups:
+        base:
+          args: |
+              [{
+                "indexkey": "OPQ128_256,IVF65536_HNSW32,PQ128x4fsr",
+                "url": "https://dl.fbaipublicfiles.com/billion-scale-ann-benchmarks/track1_baseline_faiss/bigann-1B.IVF1M_2level_PQ64x4fsr.faissindex"
```

Comment:
@zzb111  Can you please check that running bigann-1B downloads this index file and actually runs? It doesn't seem to work:

021-11-04 12:29:35,740 - annb.95019856d3 - INFO - Trying to instantiate benchmark.algorithms.faiss_t1_plus.Faiss(['euclidean', {'indexkey': 'OPQ128_256,IVF65536_HNSW32,PQ128x4fsr', 'url': 'https://dl.fbaipublicfiles.com/billion-scale-ann-benchmarks/track1_baseline_faiss/bigann-1B.IVF1M_2level_PQ64x4fsr.faissindex'}])
2021-11-04 12:29:36,118 - annb.95019856d3 - INFO - Running team11 on bigann-1B
2021-11-04 12:29:36,118 - annb.95019856d3 - INFO - Got 10000 queries
2021-11-04 12:29:36,122 - annb.95019856d3 - INFO - Build-time number of threads: 32
2021-11-04 12:29:36,126 - annb.95019856d3 - INFO - Update add-time parameters
2021-11-04 12:29:36,127 - annb.95019856d3 - INFO -    update quantizer efSearch= 16 -> 64
2021-11-04 12:29:36,127 - annb.95019856d3 - INFO -   update quantizer efConstruction= 40 -> 168
2021-11-04 12:29:36,127 - annb.95019856d3 - INFO - setting maxtrain to 3276800
2021-11-04 12:29:36,127 - annb.95019856d3 - INFO - getting first 3276800 dataset vectors for training
2021-11-04 12:29:36,127 - annb.95019856d3 - INFO - Traceback (most recent call last):
2021-11-04 12:29:36,127 - annb.95019856d3 - INFO -   File "run_algorithm.py", line 3, in <module>
2021-11-04 12:29:36,127 - annb.95019856d3 - INFO -     run_from_cmdline()
2021-11-04 12:29:36,127 - annb.95019856d3 - INFO -   File "/home/app/benchmark/runner.py", line 227, in run_from_cmdline
2021-11-04 12:29:36,127 - annb.95019856d3 - INFO -     run(definition, args.dataset, args.count, args.runs, args.rebuild,
2021-11-04 12:29:36,128 - annb.95019856d3 - INFO -   File "/home/app/benchmark/runner.py", line 94, in run
2021-11-04 12:29:36,128 - annb.95019856d3 - INFO -     algo.fit(dataset)
2021-11-04 12:29:36,128 - annb.95019856d3 - INFO -   File "/home/app/benchmark/algorithms/faiss_t1_plus.py", line 190, in fit
2021-11-04 12:29:36,128 - annb.95019856d3 - INFO -     xt2 = next(ds.get_dataset_iterator(bs=maxtrain))
2021-11-04 12:29:36,128 - annb.95019856d3 - INFO -   File "/home/app/benchmark/datasets.py", line 295, in get_dataset_iterator
2021-11-04 12:29:36,128 - annb.95019856d3 - INFO -     filename = self.get_dataset_fn()
2021-11-04 12:29:36,128 - annb.95019856d3 - INFO -   File "/home/app/benchmark/datasets.py", line 290, in get_dataset_fn
2021-11-04 12:29:36,128 - annb.95019856d3 - INFO -     raise RuntimeError("file not found")
2021-11-04 12:29:36,128 - annb.95019856d3 - INFO - RuntimeError: file not found
2021-11-04 12:29:36,590 - annb.95019856d3 - ERROR - ['euclidean', {'indexkey': 'OPQ128_256,IVF65536_HNSW32,PQ128x4fsr', 'url': 'https://dl.fbaipublicfiles.com/billion-scale-ann-benchmarks/track1_baseline_faiss/bigann-1B.IVF1M_2level_PQ64x4fsr.faissindex'}]
Trying to instantiate benchmark.algorithms.faiss_t1_plus.Faiss(['euclidean', {'indexkey': 'OPQ128_256,IVF65536_HNSW32,PQ128x4fsr', 'url': 'https://dl.fbaipublicfiles.com/billion-scale-ann-benchmarks/track1_baseline_faiss/bigann-1B.IVF1M_2level_PQ64x4fsr.faissindex'}])
Running team11 on bigann-1B
Got 10000 queries
Build-time number of threads: 32
Update add-time parameters
   update quantizer efSearch= 16 -> 64
  update quantizer efConstruction= 40 -> 168
setting maxtrain to 3276800
getting first 3276800 dataset vectors for training
Traceback (most recent call last):
  File "run_algorithm.py", line 3, in <module>
    run_from_cmdline()
  File "/home/app/benchmark/runner.py", line 227, in run_from_cmdline
    run(definition, args.dataset, args.count, args.runs, args.rebuild,
  File "/home/app/benchmark/runner.py", line 94, in run
    algo.fit(dataset)
  File "/home/app/benchmark/algorithms/faiss_t1_plus.py", line 190, in fit
    xt2 = next(ds.get_dataset_iterator(bs=maxtrain))
  File "/home/app/benchmark/datasets.py", line 295, in get_dataset_iterator
    filename = self.get_dataset_fn()
  File "/home/app/benchmark/datasets.py", line 290, in get_dataset_fn
    raise RuntimeError("file not found")
RuntimeError: file not found